contracts/rewards/AutoCompoundingRewards.sol

```diff
@@ -281,10 +298,51 @@ contract AutoCompoundingRewards is IAutoCompoundingRewards, ReentrancyGuardUpgra
         emit ProgramEnabled({ pool: pool, status: status, remainingRewards: p.remainingRewards });
     }
 
+    /**
+     * @inheritdoc IAutoCompoundingRewards
+     */
+    function setAutoTriggerCount(uint256 newAutoTriggerCount) external onlyAdmin {
+        uint256 prevAutoTriggerCount = _autoTriggerCount;
```

Comment:
`newAutoTriggerCount` should be at least greater than 0, no?

Reply:
IMO, no. It's a DAO function, so it may as well give full flexibility (in this case, a kind of 'auto-trigger disablement').

Reply:
But nevertheless... done